crates/flycheck/src/lib.rs

```diff
@@ -461,6 +470,9 @@ impl CargoActor {
         let mut read_at_least_one_stdout_message = false;
         let mut read_at_least_one_stderr_message = false;
         let process_line = |line: &str, error: &mut String| {
+            if line == "CLEAR" {
```

Comment:
I'm open to some sort of json format or something instead to be consistent to the rest of the file, but didn't bother implementing it since I don't know what the right structure for something like that would be.